components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/LoginEventHookHandler.java

```diff
@@ -0,0 +1,224 @@
+        return canHandle;
+    }
+
+    private boolean isSupportedEvent(String eventName) {
+
+        return IdentityEventConstants.EventName.AUTHENTICATION_SUCCESS.name().equals(eventName) ||
+                IdentityEventConstants.EventName.AUTHENTICATION_STEP_FAILURE.name().equals(eventName) ||
+                IdentityEventConstants.EventName.AUTHENTICATION_FAILURE.name().equals(eventName);
+    }
+
+    @Override
+    public void handleEvent(Event event) throws IdentityEventException {
+
+        EventData eventData = eventHookHandlerUtils.buildEventDataProvider(event);
+
+        if (eventData.getAuthenticationContext().isPassiveAuthenticate()) {
+            return;
+        }
+
+        //TODO: Add the implementation to read the Event Schema Type from the Tenant Configuration
+        LoginEventPayloadBuilder payloadBuilder = PayloadBuilderFactory
+                .getLoginEventPayloadBuilder(EVENT_SCHEMA_TYPE_WSO2);
+        EventAttribute loginEventPublisherConfig = getLoginEventPublisherConfigForTenant(
+                eventData.getAuthenticationContext().getLoginTenantDomain(), event.getEventName());
+        EventPayload eventPayload;
+        String eventUri;
+
+        if (IdentityEventConstants.EventName.AUTHENTICATION_SUCCESS.name().equals(event.getEventName()) &&
+                loginEventPublisherConfig.isPublishEnabled()) {
+            eventPayload = payloadBuilder.buildAuthenticationSuccessEvent(eventData);
+            eventUri = eventHookHandlerUtils.getEventUri(Constants.EventHandlerKey.LOGIN_SUCCESS_EVENT);
+            String tenantDomain = eventData.getAuthenticationContext().getLoginTenantDomain();
+            SecurityEventTokenPayload securityEventTokenPayload = eventHookHandlerUtils.buildSecurityEventToken(eventPayload, eventUri);
+            eventHookHandlerUtils.publishEventPayload(securityEventTokenPayload, tenantDomain, eventUri);
+        } else if (IdentityEventConstants.EventName.AUTHENTICATION_STEP_FAILURE.name().equals(event.getEventName()) &&
+                loginEventPublisherConfig.isPublishEnabled()) {
+            eventPayload = payloadBuilder.buildAuthenticationFailedEvent(eventData);
+            eventUri = eventHookHandlerUtils.getEventUri(Constants.EventHandlerKey.LOGIN_FAILED_EVENT);
+            String tenantDomain = eventData.getAuthenticationContext().getLoginTenantDomain();
+            SecurityEventTokenPayload securityEventTokenPayload = eventHookHandlerUtils.buildSecurityEventToken(eventPayload, eventUri);
+            eventHookHandlerUtils.publishEventPayload(securityEventTokenPayload, tenantDomain, eventUri);
+        }
+    }
+
+    private EventAttribute getLoginEventPublisherConfigForTenant(String tenantDomain, String eventName) {
+
+        if (StringUtils.isEmpty(tenantDomain) || MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+            return new EventAttribute();
+        }
```

Comment:
why we skip carbon.super ? IMO, we don't need to.
There are different ways super tenant in accessed as of now, like without /t, and with /t/carbon.super.
Both should work